e2e/poms/publishedForm.ts

```diff
@@ -1,54 +1,75 @@
-import { Page, Locator, expect } from "@playwright/test";
-import { PUBLISHED_FORM_SELECTORS } from "../constants/selectors";
+import { Page, expect } from "@playwright/test";
+import { PUBLISHED_FORM_SELECTORS } from "../constants/selectors/publishedForm";
 
-const { FORM_ELEMENTS, DROPDOWN_ELEMENTS, RESULT_ELEMENTS } = PUBLISHED_FORM_SELECTORS;
-
-export class PublishedFormPage {
-    readonly page: Page;
-    readonly emailField: Locator;
-    readonly phoneField: Locator;
-    readonly submitButton: Locator;
+export default class PublishedForm {
+    private page: Page;
 
     constructor(page: Page) {
         this.page = page;
-        this.emailField = page.getByTestId(FORM_ELEMENTS.EMAIL_FIELD);
-        this.phoneField = page.getByTestId(FORM_ELEMENTS.PHONE_NUMBER_FIELD);
-        this.submitButton = page.getByTestId(FORM_ELEMENTS.SUBMIT_BUTTON);
     }
 
-    async goto(url: string) {
-        await this.page.goto(url);
+    initialize(page: Page) {
+        this.page = page;
     }
-    
-    async fillEmail(email: string) {
-        await this.emailField.fill(email);
+
+    async getFormFields() {
+        return await this.page.getByTestId(PUBLISHED_FORM_SELECTORS.formGroupQuestion).allTextContents();
     }
```

Comment:
1. Use arrow functions
2. Avoid redundant awaits

```suggestion
    getFormFields = this.page.getByTestId(PUBLISHED_FORM_SELECTORS.formGroupQuestion).allTextContents;
```